databuilder/backends/graphnet.py

```diff
@@ -12,69 +12,14 @@
 class GraphnetBackend(BaseBackend):
     """Backend for working with data in Graphnet."""
 
-    query_engine_class = MSSQLQueryEngine
+    query_engine_class = SparkQueryEngine
     patient_join_column = "Patient_ID"
-    implements = [databuilder.tables.beta.graphnet, databuilder.tables.beta.smoketest]
+    implements = [databuilder.tables.beta.smoketest]
 
     patients = MappedTable(
         schema=SCHEMA,
         source="Patients",
```

Comment:
The schema spreadsheet has a singular `Patient` here.

Reply:
No, wait, it's got both!

![image](https://user-images.githubusercontent.com/28734/194103152-8d82ef84-46b0-4ce5-bab8-1653373f17a4.png)

@benbc can you ask Graphnet which we should use?